payments-core/src/main/java/com/stripe/android/googlepaysheet/GooglePayLauncher.kt

```diff
@@ -10,22 +10,22 @@ import kotlinx.coroutines.launch
 /**
  * A drop-in class that presents a Google Pay sheet to collect a customer's payment.
```

Comment:
Is this the thing that clients will interact with?   If so `Sheet` actually seems like the convention we have been using.  I apologize, I probably missed some previous conversations.

A couple of things that might be helpful to add into the comment.  Is this a class the client will interact with?  How is it's responsibilities different from the `GooglePayController`?

Reply:
`GooglePayLauncher` delegates to `GooglePayController` under the hood. I'm open to naming suggestions.